src/main/scala/com/cloudant/clouseau/IndexService.scala

```diff
@@ -117,6 +121,11 @@ class IndexService(ctx: ServiceContext[IndexServiceArgs]) extends Service(ctx) w
         pendingSeq = newSeq
         logger.debug("Pending sequence is now %d".format(newSeq))
         'ok
+      case SetPurgeSeqMsg(newSeq: Long) =>
+        pendingPurgeSeq = newSeq
+        logger.debug("Pending purge sequence is now %d".format(newSeq))
+        commit(pendingSeq, pendingPurgeSeq)
```

Comment:
we can safely assume clouseau crashes are rare from our production experience.

commit is expensive, the most expensive operation there is in Lucene, I think.

So the choice is between an expensive commit for every purge or a less expensive delete on retry for every clouseau crash.

We should also not say "delete index" when we really mean "delete document". :)